source/nodejs/adaptivecards-designer/src/device-emulation.ts

```diff
@@ -3,11 +3,11 @@
 
 export abstract class DeviceEmulation {
   readonly name: string;
-  readonly styleSheet: string;
+  readonly maxWidth: string | null;
```

Comment:
I know this is contentious and people tend to disagree, but... I suggest using `undefined` instead of `null` pretty much everywhere (there are still parts of the designer codebase that use null that I've wanted to change for a while, but the renderer SDK has entirely moved to `undefined`). The main reasons are:
- Uninitialized variables are `undefined`, not `null`. Using `null` thus leads to potentially having to do multiple checks to determine what to do with a variable
- `typeof null == object` whereas `typeof undefined == undefined`. So if you have say `let s: string | null = null`, even though `s` is explicitly typed as a `string` `typeof s` will return `object`. Not cool in my book.

Reply:
I have my own thoughts on the matter, but I'll gladly defer to however the maintainers of this project want to handle non-values, so I'll refactor this to use `undefined`.

Reply:
Thanks - and if you want to share your thoughts with me offline, I'd love to know.